frontend/lib/js/query-node-editor/ParameterTableFilters.js

```diff
@@ -120,7 +121,6 @@ const ParameterTableFilters = (props: PropsType) => (
                     inputType="number"
                     input={{
                       value: filter.value,
-                      defaultValue: filter.defaultValue,
```

Comment:
Why did you remove this?

Reply:
I ask myself the same thing. I did not consciously remove that.